src/components/Navigation.js

```diff
@@ -0,0 +1,36 @@
+import { Link } from 'react-router-dom';
+import style from './style/style.css';
+import planet from '../aspects/space.png';
+
+export default function Navbar() {
+  return (
+    <>
+      <div className={style.app}>
+        <nav className="navContainer">
+          <div className="title">
+            <img src={planet} alt="Space Travel Hub logo" className="logo" />
+            <h1 className="navHeader">Space Traveler&apos;s Hub</h1>
+          </div>
+          <ul className="navLinks">
+            <li>
```

Comment:
- [ ] I kindly suggest that you make use of `<NavLink>` for navigation as indicated in the requirements. Also kindly make sure that only the active page is indicated with an underline. With Navlink you can make your active link underlined very easily:
<img width="959" alt="image" src="https://user-images.githubusercontent.com/52934128/225853924-0dbd6c42-b749-44f5-a8eb-fa8492a429bb.png">

## Example of Navlink:

```
 <NavLink
      to="/rockets"
      style={({ isActive }) => ({
        textDecoration: isActive ? 'underline' : '#545e6f',
      })}
      className="navRockets"
    >
      {item.text}
  </NavLink>
```

The above is only an example and may not apply directly to your code.

Reply:
Thank you for these good suggestions.

Reply:
@otmanTR you're welcome